### PR TITLE
ticdc: add open protocol parser demo link

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Add ticdc open protocol parsing demo link.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4518
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: ticdc/ticdc-open-protocol.md [hunk 1]
```diff
@@ -270,6 +270,13 @@ COMMIT;
 14. [partition=1] [key="{\"ts\":415508881038376963,\"t\":3}"] [value=]
 ```
 
+## Protocol parsing for consumers
+
+Currently, TiCDC does not provide the standard implementation of TiCDC Open Protocol parsing but demonstrations of parsing for the Golang version and Java version. You can refer to the data format provided in this document and the following demonstrations to implement the protocol parsing for consumers.
```
Comment (yikeke) [inline ticdc/ticdc-open-protocol.md:275]:
```suggestion
TiCDC does not yet provide the standard implementation of parsing TiCDC Open Protocol for consumers. Currently it only provides the Golang and Java version of protocol parsing for demonstration. You can refer to the data format provided in this document and the following demonstrations to implement the protocol parsing for consumers.
```

**Review by yikeke — COMMENTED** [review]
@amyangfei PTAL at my suggestion below:

File: ticdc/ticdc-open-protocol.md (hunk 1 above)
Comment (amyangfei) [reply ticdc/ticdc-open-protocol.md:275]: What about
Currently, TiCDC does not provide the standard parsing library for TiCDC Open Protocol, but Golang version and Java version demonstrations are provided.

Comment (yikeke) [reply ticdc/ticdc-open-protocol.md:275]:
```suggestion
Currently, TiCDC does not provide the standard parsing library for TiCDC Open Protocol, but the Golang version and Java version of parsing demonstrations are provided. You can refer to the data format provided in this document and the following demonstrations to implement the protocol parsing for consumers.
```
LGTM @TomShawn @amyangfei

Comment (TomShawn) [reply ticdc/ticdc-open-protocol.md]: Suggestion applied.

**Review by yikeke — APPROVED** [review]

**Comment by yikeke** [comment]
LGTM

**Review by amyangfei — COMMENTED** [review]
LGTM

**Pull request merged**